### PR TITLE
Allow hiding unique portals

### PR DESCRIPTION
**Pull request by tumik**
This is @michaeldever's PR on [iitc-project/ingress-intel-total-conversion](https://github.com/iitc-project/ingress-intel-total-conversion).

[Link to original PR with screenshots and explanation](https://github.com/iitc-project/ingress-intel-total-conversion/pull/985)

### CODE REVIEW

**Comment by johnd0e** [comment]
Thank you for this contribution!

The feature itself is great, but we still need to figure out if the most proper way is used in actual implementation.

I see there are many details discussed in https://github.com/iitc-project/ingress-intel-total-conversion/issues/962.

**Comment by modos189** [comment]
I think we can accept until an improved implementation has been proposed

**Comment by johnd0e** [comment]
> I think we can accept until an improved implementation has been proposed

I am working on it.

File: plugins/uniques.user.js [hunk 1]
```diff
@@ -367,6 +373,111 @@ window.plugin.uniques.loadLocal = function(name) {
...
+    var hideAsVisited = window.plugin.uniques.hideVisitedPortals && uniqueInfo.visited;
+    var hideAsCaptured = window.plugin.uniques.hideCapturedPortals && uniqueInfo.captured;
+    return hideAsVisited || hideAsCaptured;
+  }
+  return false;
+}
+
+window.plugin.uniques.updateOtherPluginLabels = function() {
+  if (window.plugin.portalNames)
+    window.plugin.portalNames.updatePortalLabels();
```
Comment (johnd0e) [inline plugins/uniques.user.js:388]: It's not best idea to call foreign plugins functions directly.
AFAIK, this patch was made before needed hooks were implemented (https://github.com/iitc-project/ingress-intel-total-conversion/pull/977)

**Comment by johnd0e** [comment]
BTW, there are good alternatives: https://github.com/xificurk/iitc-plugins#highlight-unique-visitscaptures-using-opacity-install

**Pull request closed without merging**